### ISSUE TITLE
bower.json in https://github.com/ezraroi/ngJsTree/archive/v0.0.4.tar.gz is wrong.

### ISSUE THREAD

**Issue by jimklo** [ezraroi/ngJsTree#12]
the archive distribution used by bower for v0.0.4 contains the version number 0.0.3:

```
{
    "name": "ng-js-tree",
    "main": "dist/ngJsTree.js",
    "version": "0.0.3",
    "homepage": "https://github.com/ezraroi/ngJsTree",
    "authors": [
        "ezraroi <ezraroi@gmail.com>"
    ],
    "description": "Angular directive for the famous JsTree",
    "keywords": [
        "Angular",
        "JsTree"
    ],
    "dependencies": {
        "angular": "~1.3",
        "jstree" : "~3.0.8"
    },
    "license": "MIT",
    "ignore": [
        "**/.*",
        "node_modules",
        "bower_components",
        "test",
        "temp",
        "demo",
        "Gruntfile.js",
        "package.json",
        "README.md"
    ],
    "devDependencies": {
        "angular-mocks": "~1.3.0",
        "jquery": "~2.1.1"
    }
}
```

**Comment by ezraroi** [comment]
You are right. Will be fixed in the next version.

Thanks

**Comment by windmaomao** [comment]
npm http GET https://registry.npmjs.org/ng-js-tree
npm http 404 https://registry.npmjs.org/ng-js-tree
npm ERR! 404 'ng-js-tree' is not in the npm registry.

i couldn't find ng-js-tree, did i miss type anything

npm install ng-js-tree --save

**Comment by windmaomao** [comment]
my god, it's bower not npm.

**Comment by ezraroi** [comment]
Fixed, will be released in 0.0.5 version
